pkg/sqlstore/store_user.go

```diff
@@ -17,6 +19,53 @@ type UserRows struct {
 	Rows    [][]interface{} `json:"rows"`
 }
 
+// UserData wraps data from the db that may be raw json or any other value.
+type UserData struct {
+	jsonValue  json.RawMessage
+	otherValue interface{}
+}
+
+// Value returns the underlying value.
+func (u *UserData) Value() interface{} {
+	if u.jsonValue != nil {
+		return u.jsonValue
+	}
+	return u.otherValue
+}
+
+// Scan implements Scan.
+func (u *UserData) Scan(src interface{}) error {
+	u.jsonValue = nil
+	u.otherValue = nil
+	switch src := src.(type) {
+	case string:
```

Comment:
In this case, the string might still contain a valid JSON value that isn't an object or array, right? So an int will end up being returned as a string? That is probably ok, but just wanted to confirm.